strawberry/types/types.py

```diff
@@ -125,7 +136,7 @@ def is_implemented_by(self, root: Union[type, dict]) -> bool:
         # TODO: Accept StrawberryObject instead
         # TODO: Support dicts
         if isinstance(root, dict):
-            raise NotImplementedError()
+            raise NotImplementedError
```

Comment:
👀

Reply:
I think my `ruff` autoformatter did this =P. It is a flake8 plugin rule that it might have implemented in a recent version

Reply:
ah, nice :D